omeroweb/webclient/tree.py

```diff
@@ -753,31 +804,20 @@ def marshal_images(
     )
 
     for e in qs.projection(q, params, service_opts):
-        e = unwrap(e)[0]
-        d = [
-            e["id"],
-            e["archived"],
-            e["name"],
-            e["ownerId"],
-            e["image_details_permissions"],
-            e["filesetId"],
-        ]
-        kwargs = {"conn": conn, "row": d[0:6]}
-        if load_pixels:
-            d = [e["sizeX"], e["sizeY"], e["sizeZ"], e["sizeT"]]
-            kwargs["row_pixels"] = d
+        data = unwrap(e)[0]
+        kwargs = {}
         if date:
-            kwargs["acqDate"] = e["acqDate"]
-            kwargs["date"] = e["date"]
+            kwargs["acqDate"] = data["acqDate"]
+            kwargs["date"] = data["date"]
 
         # While marshalling the images, determine if there are any
         # images mentioned in shares that are not in the results
         # because they have been deleted
-        if share_id is not None and image_rids and e["id"] in image_rids:
+        if share_id is not None and image_rids and data["id"] in image_rids:
             image_rids.remove(e["id"])
```

Comment:
```suggestion
            image_rids.remove(data["id"])
```

should fix the failing integration test - https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/158/testReport/OmeroWeb.test.integration.test_tree/TestTree/test_marshal_images_share/